modules/persistent-storage-csi-snapshots-operator.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-snapshots.adoc
+
+[id="csi-snapshots-operator_{context}"]
+= About the CSI Snapshot Controller Operator
+
+The CSI Snapshot Controller Operator (cluster-csi-snapshot-controller-operator) is installed by the Cluster Version Operator (CVO) in all clusters by default.
+
+The CSI Snapshot Controller Operator deploys both the csi-snapshot-controller and the csi-external-snapshotter sidecar into the control plane. It also deploys the Container Storage Interface (CSI) driver.
```

Comment:
This isn't entirely accurate. The CSI Snapshot Controller Operator is installed by CVO, and lives in `openshift-csi-snapshot-controller-operator`.

The Operator installs the CSI Snapshot Controller, which lives in `openshift-csi-snapshot-controller`. This is what has the controller and the sidecar. Note that it doesn't install any drivers by itself, it just watches for Snapshots.